toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/AuthenticatorState.kt

```diff
@@ -385,20 +385,21 @@ private class AuthenticatorStateImpl(
      * @since 200.2.0
      */
     private suspend fun awaitCertificateChallengeResponse(): NetworkAuthenticationChallengeResponse {
-        val selectedAlias = suspendCancellableCoroutine<String?> { continuation ->
+        val selectedAlias = safeSuspendableCancellable<String?> { safeContinuation ->
```

Comment:
I see a warning here about redundant type argument:
```suggestion
        val selectedAlias = safeSuspendableCancellable { safeContinuation ->
```